include/bluetooth/mesh/light_xyl_srv.h

```diff
@@ -111,25 +111,25 @@ struct bt_mesh_light_xyl_srv_handlers {
 	 *
 	 * @param[in] srv Server the Range state was changed on.
 	 * @param[in] ctx Context of the set message that triggered the update.
-	 * @param[in] old The old Range.
-	 * @param[in] new The new Range.
+	 * @param[in] old_range The old Range.
+	 * @param[in] new_range The new Range.
 	 */
 	void (*const range_update)(struct bt_mesh_light_xyl_srv *srv,
 				   struct bt_mesh_msg_ctx *ctx,
-				   const struct bt_mesh_light_xy_range *old,
-				   const struct bt_mesh_light_xy_range *new);
+				   const struct bt_mesh_light_xy_range *old_range,
+				   const struct bt_mesh_light_xy_range *new_range);
```

Comment:
Should also be changed in the source file, even though it doesn't have any compilation issues

Reply:
I guess it doesn't matter, those files are compiled with c compiler

Reply:
Some static tools complain if the source and header use different names though.

Reply:
This one is not used anywhere in NCS yet.

Reply:
It doesn't matter for compilation, but it'd be good practice, which is why some linters fail if we don't

Reply:
C is not subset of C++, I don't see a point trying to align C code to C++. If linter complains about that then it's either linter issue or its configuration.

Reply:
It's not about syntax or compliance, it's about maintainability. 

We've spent more time on this comment than it's really worth, but the point is that we want to keep the same signature in the header and source file, as this makes it easier to maintain, search for and talk about. The meaning of the name isn't lost here, so it's more of a matter of principle.

Reply:
It seems correct in the source code:
https://github.com/nrfconnect/sdk-nrf/blob/master/subsys/bluetooth/mesh/light_xyl_srv.c#L331
https://github.com/nrfconnect/sdk-nrf/blob/master/subsys/bluetooth/mesh/light_xyl_srv.c#L331

Reply:
Well, what the hell. I'm sorry, looks like it does :smile: Let's get this merged then! :slightly_smiling_face: